Update supported local database provider (#4998)
umbraco/UmbracoDocs: umbraco-cloud/databases/local-database.md

When cloning down your project to work locally you might want to have a look in your database every now and then.

Since Umbraco 10, SQL CE is no longer supported, instead, Umbraco now comes with SQLite out of the box. When you clone down your Umbraco project and restore its content, it will create a `Umbraco.sqlite.db` file.

> To view your local database, you will need to use a program like [DB Browser for SQLite](https://sqlitebrowser.org/) which lets you view and query your local SQLite database.

Author: ```suggestion
To view your local SQLite database, you will need to use a program like [DB Browser for SQLite](https://sqlitebrowser.org/) or a Visual Studio extension like [SQLite and SQL Server Compact Toolbox](https://marketplace.visualstudio.com/items?itemName=ErikEJ.SQLServerCompactSQLiteToolbox).
```

> You can also configure your project to prefer SQL Server LocalDb when it's available on your local machine by enabling the Deploy `PreferLocalDbConnectionString` setting.

Author: This needs a link to https://docs.umbraco.com/umbraco-deploy/deploy-settings#preferlocaldbconnectionstring and maybe https://docs.umbraco.com/umbraco-deploy/upgrading/version-specific#database-initialization for more detailed information about this change.

Reviewer: Sounds like a good idea with adding a link to the deploy setting section, my suggestion would to make the `PreferLocalDbConnectionString` text into a link :) 

Would you be up for doing that @ronaldbarendse?

Author: I asked because I wasn't sure how to add internal links (especially since it's within another, version specific, GitBook space)... Will this work?
```suggestion
You can also configure your project to prefer SQL Server LocalDb when it's available on your local machine by enabling the Deploy [`PreferLocalDbConnectionString`](../../10/umbraco-deploy/deploy-settings.md#preferlocaldbconnectionstring) setting.
```

Reviewer: Ahh fair :D If you can add it like this: _**[`PreferLocalDbConnectionString`](https://docs.umbraco.com/umbraco-deploy/deploy-settings#preferlocaldbconnectionstring)**_ that would be perfect, then it will open up in a new tab 😄

Author: ```suggestion
You can also configure your project to prefer SQL Server LocalDb when it's available on your local machine by enabling the Deploy [`PreferLocalDbConnectionString`](https://docs.umbraco.com/umbraco-deploy/deploy-settings#preferlocaldbconnectionstring) setting.
```
